docs/azure/sdk/snippets/protocol-convenience-methods/SCMProtocol/Program.cs

```diff
@@ -1,34 +1,34 @@
-﻿using OpenAI;
-using OpenAI.Chat;
+﻿using OpenAI.Chat;
 using System.ClientModel;
 using System.ClientModel.Primitives;
 using System.Text.Json;
 
 // Create the client
-OpenAIClient client = new("<your-openai-api-key>");
-ChatClient chatClient = client.GetChatClient("gpt-4");
+ChatClient client = new(
+    model: "gpt-4o-mini",
+    credential: Environment.GetEnvironmentVariable("OPENAI_API_KEY")!);
 
 // Create the request content
 BinaryData input = BinaryData.FromBytes("""
-    {  
-        "model": "gpt-4o",
+    {
+        "model": "gpt-4o-mini",
         "messages": [
            {
                "role": "user",
-               "content": "What is Microsoft Azure?."
+               "content": "What is Microsoft Azure?"
            }
         ]
     }
     """u8.ToArray());
 using BinaryContent content = BinaryContent.Create(input);
 
 // Call the protocol method
-ClientResult result = chatClient.CompleteChat(
-        content,
-        new RequestOptions()
-        {
-            ErrorOptions = ClientErrorBehaviors.NoThrow
-        });
+ClientResult result = client.CompleteChat(
+    content,
+    new RequestOptions
+    {
+        ErrorOptions = ClientErrorBehaviors.NoThrow,
+    });
 
 PipelineResponse response = result.GetRawResponse();
```

Comment:
It might be worth noting here that you can still call `ToDynamicFromJson` with SCM-based responses if desired.  This API lives in Azure.Core so an app would need to reference this package, but nothing prevents someone from mixing Azure and unbranded code from an application usage perspective -- in fact, it is expected that people will do this.